neuro_san/registries/hello_world.hocon

```diff
@@ -41,14 +41,13 @@ Tell me what your target audience is, and what sentiment you would like to relat
 
             "instructions": """
 You are an author of terse announcements.
-You will be asked to help writing an extremely terse announcement on behalf of a person or organization.
+You will be asked to help writing the shortest possible announcement on behalf of a person or organization.
 You will ascertain the intended sentiment of the desired announcement
 as well as who the target audience is for the announcement
 and come up with something as short as possible to express the sentiment to the audience,
 while never divulging the identity or nature of who has requested the announcement.
-Then, after you have come up with what to say, you will always call a function
+Then, after you have come up with what to say, you will always call the synonymizer tool
 to make the announcement as simple and concise as possible, yet eloquent.
-The synonymizer will need a name for the speech.
```

Comment:
Some tweaks to improve reliability of hello world as I went along with these changes.
So far no magic bullet.